### ISSUE TITLE
pre-commit flake8 doesn't work

### ISSUE THREAD

**Issue by churnikov** [OML-Team/open-metric-learning#212]
I've installed pre-commit hooks and see the following
<img width="617" alt="image" src="https://user-images.githubusercontent.com/8545082/199214104-a7da9b7f-cfaa-4c67-bf13-051bba7de62f.png">

But I knew that I had errors, that flake8 should pickup, so I installed flake8 separately and saw the following (screenshot doesn't have my errors):

<img width="911" alt="image" src="https://user-images.githubusercontent.com/8545082/199214461-bea90479-dc49-4ce7-b082-cd7ad10049fe.png">

I think that flake8 should be fixed in pre-commit config file

**Comment by AlekseySh** [comment]
@churnikov You are right! Thank you for finding that! We will change the config for our hook and push the updates to the repo. Then you can pull it.

**Comment by AlekseySh** [comment]
@churnikov we fixed that

**Comment by DaloroAT** [comment]
@churnikov thank you
The problem was in pattern for files

**Comment by churnikov** [comment]
No problem :)

I thought that that might be a problem. Glad to help out 😊
